cdap-ui/app/directives/complex-schema/complex-schema.js

```diff
@@ -68,6 +68,32 @@ function ComplexSchemaController (avsc, SCHEMA_TYPES, $scope, uuid, $timeout, Sc
     field.nested = SchemaHelper.checkComplexType(field.displayType);
   };
 
+  vm.pasteFields = (event, field) => {
+
+    let pastedData = event.clipboardData.getData('text/plain');
+    let pastedDataArr = pastedData.replace(/[\n\r\t ]/g, '$#$').split('$#$');
```

Comment:
Please also add "," and "|" in the list of splitters. It would be really nice if we can just paste the headers of the csv file and generate schema from that